security/secrets.md

```diff
@@ -0,0 +1,59 @@
+# Secrets Management
+
+If secrets (e.g. API keys, database logins, or other credentials) are to be stored or shared, they MUST be stored or shared in one secure place, to signify one source of truth and to make resources easier to find.
+
+[AWS Parameter Store](http://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html) SHOULD be used to store secrets.
+
+## Parameter Store
+
+### Name conventions
+
+Parameter names MUST use a hierarchy and list the environment name and other descriptors:
+
+```
+environment/{service or application name}/{other descriptors}
```

Comment:
Not to chime in at the last minute, can we tweak the convention for parameter naming a bit?
I agree that the environment should be the first part of the segment....can we say:

The name MUST include the following:

* The first segment of the parameter name MUST be environment/stage/tier if applicable.
* The service that this is a parameter for.
* The app that uses this (I could see this being omitted if it's a resource shared by multiple apps).
* A description of what this is.

Examples may be:

* `/production/mms/rds/password-for-mms_productions-root-user`
* `/production/elasticcache/my-cluser-name/password`
* `/loggly/my-loggly-user-name/api-key` (environment/stage/tier isn't applicable)

Reply:
I don't care much about the order of the segments - but think it would be good to see examples of good things to see in the segments.

Reply:
I think this is great! 👍

Reply:
Would you mind submitting a PR for this update? 😬

Reply:
I made the changes you suggested @nodanaonlyzuul and merged the PR.

Thanks everyone!